src/components/common/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -214,12 +214,12 @@ class BaseCourseCard extends Component {
     return null;
   };
 
-  renderChidren = () => {
+  renderChildren = () => {
     const { children } = this.props;
     if (children) {
       return (
         <div className="row">
-          <div className="col mb-3">
+          <div className="col">
```

Comment:
By having the class `.mb-3` here, there is unnecessary margin above the misc course text at the bottom of in-progress course cards that have no notifications (due dates). Instead, I'm moving the `.mb-3` class to the `Notifications` component instead, fixing the extra margin.